csuntechlab/iSTART
# STR-15 -- FE: Setup Login

# Description
'set up login using backend route and cookies, changed vuex file structure, added component and vuex tests and their respective file structure'
  
# Testing
1. open localhost:8080
2. to login correctly: 
    CSUN Credentials: steve@csun.edu
    password: none
2.5. login incorrectly, you will get an alert
3. should take you to welcome page with a message and option to logout
4. reload and should redirect you to welcome page, same when you close browser
5. try to access login route, should redirect to welcome
6. logout, should redirect you back to login page
7. run yarn test:unit to run front end tests


# Installation
'no'
